homeassistant/components/saj/sensor.py

```diff
@@ -182,6 +189,14 @@ def state(self):
         """Return the state of the sensor."""
         return self._state
 
+    @property
+    def device_state_attributes(self):
+        """Return extra attributes of the sensor."""
+        return {
+            ATTR_INVERTER_NAME: self._inverter_name,
+            ATTR_INVERTER_SN: self._serialnumber,
```

Comment:
We don't add static info to the state attributes. The state attributes are meant for dynamic info related to the state of the entity or info that is essential for automations.

Reply:
Thank you for explaining, wasn't aware of this.
Changes pushed.